script/lib/check-aws-credentials

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+red='\033[0;31m'
+plain='\x1B[0m'
+
+STATUS=$(aws sts get-caller-identity --profile composer 2>&1 || true)
+if [[ ${STATUS} =~ (ExpiredToken) ]]; then
+echo -e "${red}Credentials for the composer profile are expired. Please fetch new credentials and run this again.${plain}"
```

Comment:
This is cool. Are these cross emojis? Would it run on all terminals?

Reply:
This should be fine everywhere that supports [unicode 1.0](https://emojipedia.org/emoji-1.0/) (the link has a big list of what's included), which is now widely used. (This can matter when we publish content with emojis – we've had issues in Composer where newer emojis don't appear on all platforms!)